backend/googlecloudstorage/googlecloudstorage.go

```diff
@@ -799,6 +831,18 @@ func (f *Fs) makeBucket(ctx context.Context, bucket string) (err error) {
 // to delete was not empty.
 func (f *Fs) Rmdir(ctx context.Context, dir string) (err error) {
 	bucket, directory := f.split(dir)
+	// Remove directory marker file
+	if f.opt.DirectoryMarkers && bucket != "" && directory != "" {
+		markerFilePath := fmt.Sprintf("%s/", directory)
+		if f.rootDirectory != "" {
+			markerFilePath = fmt.Sprintf("%s/%s", f.rootDirectory, markerFilePath)
+		}
+		markerFileObject := &Object{
+			fs:     f,
+			remote: markerFilePath,
+		}
+		markerFileObject.Remove(ctx)
```

Comment:
[linter wants you](https://github.com/rclone/rclone/runs/3263796986?check_suite_focus=true#step:14:23) to:
```
_ = markerFileObject.Remove(ctx)
```